docs/Installation/Build-From-Source.md

```diff
@@ -0,0 +1,99 @@
+description: Building Orion from source code
+<!--- END of page meta data -->
+
+# Build from Source
+
+## Prerequisites
+
+* [Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+
+!!!important
+    Orion requires Java 8+ to compile; earlier versions are not supported.
+
+* [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/)
+
+* [libsodium](Dependencies.md)
+
+## Installation on Linux / Unix / Mac OS X
+
+###Clone the Orion Repository
+
+Clone the **PegaSysEng/orion** repository:
+
+```bash
+$ git clone --recursive https://github.com/PegaSysEng/orion.git
```

Comment:
We have to decide if we use dollar sign in front of bash commands or not. My guess it that now that we have copy button, we should not.

Also we should make the cloning process consistent across our docs, see https://github.com/PegaSysEng/orion/pull/210/files#diff-539bae1eee80c23e90c720d728d42ec7R10

If I had to choose one, I prefer the https scheme.

Reply:
Fixed